Game/Source/GDKTestGyms/Public/TestGymsReplicationGraph.h

```diff
@@ -20,6 +20,7 @@ enum class EClassRepNodeMapping : uint32
 {
 	NotRouted,						// Doesn't map to any node. Used for special case actors that handled by special case nodes (UTestGymsReplicationGraphNode_PlayerStateFrequencyLimiter)
 	RelevantAllConnections,			// Routes to an AlwaysRelevantNode or AlwaysRelevantStreamingLevelNode node
+	AlwaysReplicate,				// Always replicated to Spatial
```

Comment:
Can you flesh the comment out a bit about why this is different to RelevantAllConnections